src/inspector_agent.cc

```diff
@@ -440,8 +461,10 @@ bool Agent::StartIoThread() {
 }
 
 void Agent::Stop() {
-  if (io_ != nullptr)
+  if (io_ != nullptr) {
     io_->Stop();
+    io_.reset();
```

Comment:
I have trouble understanding Node core C++ conventions, does it makes sense that `Stop()` is upper case, and `reset()` is lower case @addaleax ?